.eslintignore

```diff
@@ -0,0 +1,8 @@
+# Ignore all folders
+**/
+.**
+README.md
```

Comment:
I think ignoring everything should be as simple as `**` without the need to define these extra files and folders.

Could you please try this?
```suggestion
# Ignore all folders
**
# Allow only certain folders and file types
!components/**
!stories/**
```

For the prettierignore file, we might even want to also add `!**/*.md` to include any markdown file (after atj pointed out that prettier works with markdwon too).
Perhaps that is something we want?

Reply:
Also we can adjust this ignore file as we go along. As long as for now it does everything we want it to do.